src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -1403,6 +1432,7 @@ internal static PublishPackageViewModel FromLocalPackage(DynamoViewModel dynamoV
                 CopyrightHolder = pkg.CopyrightHolder,
                 CopyrightYear = pkg.CopyrightYear,
                 IsPublishFromLocalPackage = true,
+                CurrentPackageDirectory = pkg.RootDirectory,
```

Comment:
do we not have access to `pkg.RootDirectory` at some point? IE do we need this new property?

Reply:
I tried to not introduce a new property, but this was required in `RefreshPackageContents()` method, which is called from many places where we cannot pass a root directory from..so had to add this.